src/protocol/rand_os.c

```diff
@@ -31,10 +31,20 @@
 #if defined(__WIN32__) || defined(WIN32) || defined(__CYGWIN32__)
 #include <windows.h>
 #include <wincrypt.h>
+#elif defined(LEDGER_VAULTAPP)
+#include "os.h"
+#include "cx.h"
 #else
 #include <unistd.h>
 #include <fcntl.h>
 #include <errno.h>
+#if defined(LEDGER_VAULTHSM)
+/* #include <bolos_config.h> */
```

Comment:
I believe that those commented lines could be removed

Reply:
Removed.